css/ampdoc.css

```diff
@@ -732,22 +267,6 @@ body:not(.i-amphtml-subs-ready) [subscriptions-action],
 }
 
 
-/**
- * Forms error/success messaging containers should be hidden at first.
- */
-form [submitting],
-form [submit-success],
-form [submit-error] {
-  display: none;
-}
-
-/**
- * Form validation error messages should be hidden at first.
- */
-[visible-when-invalid]:not(.visible) {
-  display: none;
-}
-
```

Comment:
Why is `amp-live-list` staying here? And further down `amp-experiment` and `amp-share-tracking`?